rpcs3/Emu/System.cpp

```diff
@@ -509,6 +511,13 @@ void Emulator::Load(bool add_only)
 #endif
 
 		LOG_NOTICE(LOADER, "Used configuration:\n%s\n", g_cfg.to_string());
+		
+		// Set RTM usage
+		g_use_rtm = utils::has_rtm() && (utils::has_mpx() || g_cfg.core.enable_TSX);
```

Comment:
In order to make this universal you should use:
`g_use_rtm = utils::has_rtm() && g_cfg.core.enable_TSX`